### DISCUSSION TITLE
use fzf when displaying different lists eg bookmarks?

### DISCUSSION THREAD

**Discussion by tkossak** [leo-arch/clifm#270] [Ideas]
Hello

It would be nice if we could search different lists, that clifm presents, using fzf eg. when I type `bm` and I have tens of different bookmarks, it would be faster for me to just type part of the name and quickly cd into the bookmark.

What do you think?

**Comment by leo-arch** [comment]
Hi @tkossak. When you type `bm <TAB>` you should usually see the list of available bookmarks displayed via `fzf`. If not, it might be a config issue. What have you tried so far?

**Comment by tkossak** [comment]
I didn't know, it works perfectly, thank you :)
